schema/Claim.schema.yaml

```diff
@@ -0,0 +1,41 @@
+title: Claim
+'@id': stencila:Claim
+extends: Entity
+role: secondary
+status: unstable
+description: A claim represents specific reviewable facts or statements.
+$comment: |
+  It is usually displayed as a block element and can have a separate title.
+  In the context of a scholarly article this could be a theorem or proof.
+  See also
+  - [Schema.org `Claim`](https://schema.org/Claim)
+  - [JATS `<statement>`](https://jats.nlm.nih.gov/publishing/tag-library/1.2/element/statement.html)
+properties:
+  title:
+    '@id': stencila:title
```

Comment:
The `@id` property provides a mapping between the terms in this schema and those in schema.org (where appropriate). We check for conflicting mappings in `npm test`:

```
🚨 ERROR schema Property "title" is associated with more than one @id "schema:headline", "stencila:title"
```

https://dev.azure.com/stencila/stencila/_build/results?buildId=11381&view=logs&j=58ad7040-9663-5369-9bcd-5f12d621aae7&t=f3a37832-3d6f-510f-b5b1-7bcb21eda8b5&l=26

This problem will go away by extending `CreativeWork` (since that already has a `title` property mapped to `schema.org` headline) and deleting this property.

Reply:
Done and tests passing now (slowly getting closer to how these things map together ...)!